iris_grib/tests/results/integration/load_convert/sample_file_loads/lambert_grib1.cml

```diff
@@ -16,16 +16,16 @@
         <auxCoord id="61bde96d" long_name="originating_centre" points="[	US National Weather Service, National Centres for Environmental Prediction]" shape="(1,)" units="Unit('no_unit')" value_type="string"/>
       </coord>
       <coord datadims="[1]">
-        <dimCoord id="62845c7c" points="[-2396487.0124, -2392487.0124, -2388487.0124,
```

Comment:
I see that the projected transforms (here + polar-stereo) have got changed coordinate values.  I think this is ok/correct -- just a scaling factor due to changed earth radius.

Reply:
Yes, I was under a similar impression